libsolidity/grammar.txt

```diff
@@ -43,13 +43,13 @@ StorageLocation = 'memory' | 'storage'
 Block = '{' Statement* '}'
 Statement = IfStatement | WhileStatement | ForStatement | Block |
             ( PlaceholderStatement | Continue | Break | Return |
-              Throw | SimpleStatement | ExpressionStatement ) ';'
+              Throw | SimpleStatement ) ';'
 
 ExpressionStatement = Expression | VariableDefinition
```

Comment:
So should we have

`ExpressionStatement = Expression`

and

`SimpleStatement = VariableDefinition | ExpressionStatement`

again ?